libavcodec/dxva2_av1.c: fixed global motion params

tong1wu opened the pull request:
libavcodec/dxva2_av1.c:fix global motion params

    Defined in spec 5.9.24/5.9.25. Since function void
    global_motion_params(AV1DecContext *s) already updates
    gm type/params, the wminvalid parameter only need to get
    the value from cur_frame.gm_invalid.

tong1wu commented:
@xuguangxin @xhaihao @feiwan1 
please help me to review this patch

xuguangxin commented on libavcodec/dxva2_av1.c:
```diff
@@ -139,8 +139,8 @@ static int fill_picture_parameters(const AVCodecContext *avctx, AVDXVAContext *c
-        pp->frame_refs[i].wminvalid = (h->cur_frame.gm_type[AV1_REF_FRAME_LAST + i] == AV1_WARP_MODEL_IDENTITY);
-        pp->frame_refs[i].wmtype    = h->cur_frame.gm_type[AV1_REF_FRAME_LAST + i];
+        pp->frame_refs[i].wminvalid = h->cur_frame.gm_invalid[AV1_REF_FRAME_LAST + i];
+        pp->frame_refs[i].wmtype = h->cur_frame.gm_type[AV1_REF_FRAME_LAST + i];
```
why did we include this line ? It's not changed

tong1wu replied:
@xuguangxin hello guangxin, I just fixed it, that line is excluded.

tong1wu committed:
avcodec/dxva2_av1.c: fix global motion params

Defined in spec 5.9.24/5.9.25. Since function void
global_motion_params(AV1DecContext *s) already updates
gm type/params, the wminvalid parameter only need to get
the value from cur_frame.gm_invalid.

Signed-off-by: Tong Wu <tong1.wu@intel.com>

xhaihao commented:
See 
https://github.com/intel-media-ci/cartwheel-ffmpeg/blob/master/patches/0093-avcodec-dxva2_av1-fix-global-motion-params.patch

xuguangxin commented:
@xhaihao @tong1wu , why we can't upstream this patch?

xhaihao commented:
See Tong's explanation https://github.com/intel-media-ci/cartwheel-ffmpeg/pull/54#issuecomment-914743035
